colearn_keras/keras_learner.py

```diff
@@ -118,10 +118,11 @@ def vote(self, new_score) -> bool:
         :param new_score: Proposed score
         :return: bool positive or negative vote
         """
+
         if self.minimise_criterion:
-            return new_score <= self.vote_score
+            return new_score < self.vote_score
```

Comment:
unrelated, but looks like it is a bug

Reply:
It was a deliberate decision, but it might not be the correct one! I put it that way because otherwise it more quickly gets stuck in the state where no updates are accepted. But I think this way it better.